parm/atm/obs/testing/sondes.yaml

```diff
@@ -0,0 +1,106 @@
+obs space:
+  name: sondes
+  obsdatain:
+    engine: 
+      type: H5File
+      obsfile: sondes_obs_$(OBS_DATE).nc4
+      obsgrouping:
```

Comment:
same question, is this the right indentation for obsgrouping? (not 100% sure)

Reply:
@CoryMartin-NOAA I think so. A similar indentation is used in other YAML.

Reply:
@CoryMartin-NOAA done, thanks.